cadquery/occ_impl/geom.py

```diff
@@ -143,9 +143,31 @@ def distanceToPlane(self):
         raise NotImplementedError(
             "Have not needed this yet, but FreeCAD supports it!")
 
-    def projectToPlane(self):
-        raise NotImplementedError(
-            "Have not needed this yet, but FreeCAD supports it!")
+    def projectToPlane(self, *args):
```

Comment:
It would be better to support only Plane input. The user can always construct a Plane object from two Vector objects if he needs to.

Reply:
Agreed.